source/includes/language-compatibility-table-python.rst

```diff
@@ -7,459 +7,69 @@ Python 3 Compatibility
    :class: compatibility-large
 
```

Comment:
S: the lines are out of reach for commenting, but we can remove the Python 3 Compatibility header now that there is only one table